src/ISA/ISA.Spreadsheet/ArcTable.fs

```diff
@@ -0,0 +1,53 @@
+﻿module ISA.Spreadsheet.ArcTable
+
+open ISA
+open FSharpAux
+open FsSpreadsheet
+
+let annotationTablePrefix = "annotationTable"
+
+let groupColumnsByHeader (columns : list<FsColumn>) = 
+    columns
+    |> List.groupWhen (fun c ->         
+        ISA.Regex.tryParseTermAnnotation c.[0].Value 
+        |> Option.isSome
+        ||
+        c.[0].Value = "Unit"
+    )
+
+/// Returns the protocol described by the headers and a function for parsing the values of the matrix to the processes of this protocol
+let tryFromFsWorksheet (sheet : FsWorksheet) =
+    let annotationTable =
+        sheet.Tables
+        |> Seq.tryFind (fun t -> t.Name.StartsWith annotationTablePrefix)
+    match annotationTable with
+    | Some t -> 
+        let compositeColumns = 
```

Comment:
This function would be nice to parse Excel tables to ArcTable. So maybe pull it outside for access.

Reply:
I actually did mean the stuff creating `compositeColumns`, but good to have the other one too^^

Reply:
Oh :D
It's kinda hard to see in the snippets github creates

Reply:
Hmm not so sure about this, the step here is basically just a composition of two other functions. But will try to find a sensible name